Feature: Allow configuration of font size used for markdown rendering

pd95 opened the pull request:
The font size used to render the `MessageCellContent` can be configured in the settings screen:

<img width="754" alt="Screenshot of the font size setting" src="https://github.com/user-attachments/assets/9b12fa5e-02c9-4151-9ad1-2bb4b04936f7">

pd95 commented:
@kevinhermawan : I'm not sure if you have already something like this feature "in preparation", and I'm not sure whether you like strangers to start amending your code base... so I created this draft PR for discussion.

This is just some feature I was missing, so I did it myself...

pd95 commented:
Might be an implementation for #35. But no keyboard shortcuts have been added so far.

kevinhermawan commented:
> @kevinhermawan : I'm not sure if you have already something like this feature "in preparation", and I'm not sure whether you like strangers to start amending your code base... so I created this draft PR for discussion.
> 
> This is just some feature I was missing, so I did it myself...

I have a plan for this but haven't started it yet. PRs are always open to everyone, so don't hesitate. Let's make it happen, friend!

kevinhermawan commented on Ollamac/Views/Subviews/MessageCellContent.swift:
```diff
@@ -11,7 +11,9 @@ import ViewState
     @Environment(\.colorScheme) private var colorScheme
-    
+   
+    @AppStorage("defaultFontSize") private var fontSize: Double = NSFont.systemFont(ofSize: 0).pointSize
```
This is inconsistent; use value from `Defaults` instead.

kevinhermawan commented:
I've been rewriting the core of Ollamac. Sorry for the conflict, @pd95. Don't worry, it's my job to resolve. 😂

pd95 commented on SettingsModule/Sources/SettingsModule/Subviews/DefaultFontSizeField.swift:
```diff
@@ -0,0 +1,67 @@
+//
```
You are right. I adjusted it accordingly. But the problem is that `onSubmit` (of the `TextField`) is only executed if you press "Return". If you "tab out" of the field, the value is not checked. So basically you can now set a font size of 1 pt. So, in my opinion it's still better to observe changes of `fontSize` and adjust the value accordingly.

pd95 commented:
I've tried to merge with main... and to avoid "collateral damage" I've put up another branch/PR #93 to allow comparison.

pd95 commented:
I have updated this PR to the latest 3.0.1 state and fixed the issues. 
Back to "Ready for review".

Would be good if some other users would test it. If no font-size has been stored in `UserDefaults` previously we use the systems default font size. (=`NSFont.systemFontSize`)

kevinhermawan commented:
Hi @pd95, I've been testing this, and it’s working as expected. It will be merged after the conflict is resolved, thanks!

pd95 commented:
Merged and still functional 😅
